src/main/java/frc/teleop/TeleopDriver.java

```diff
@@ -30,6 +30,8 @@ public static TeleopDriver getInstance() {
     private TeleopDriver() {
         this.drive = Drive.getInstance();
         this.intake = Pickup.getInstance();
+
+        // HI MY NAME IS NOAH!
```

Comment:
Why this comment? No need please remove.

Reply:
Ok Fixed

Reply:
cool